libvirt.go

```diff
@@ -294,6 +299,24 @@ func (d *Driver) Create() error {
 	return d.Start()
 }
 
+func createImage(src, dst string) error {
+	// #nosec
```

Comment:
Which security warning is this turning off? Might be better to be specific about it.

Reply:
done